manifests/init.pp

```diff
@@ -159,6 +168,7 @@
   $rdeck_home                   = $rundeck::params::rdeck_home,
   $rdeck_config_template        = $rundeck::params::rdeck_config_template,
   $file_keystorage_keys         = $rundeck::params::file_keystorage_keys,
+  $manage_default_policy        = $rundeck::params::manage_default_policy,
```

Comment:
It looks like this is being used for both the api and acl policies. We should change this to policies, or add another variable for api.